winpr/libwinpr/crypto/cipher.c

```diff
@@ -580,18 +343,6 @@ mbedtls_cipher_type_t winpr_mbedtls_get_cipher_type(int cipher)
 		case WINPR_CIPHER_AES_256_CCM:
 			type = MBEDTLS_CIPHER_AES_256_CCM;
 			break;
-
```

Comment:
why remove these?

Reply:
Same as above, deprecated and unused cipher mapping